autogpt_platform/backend/backend/blocks/airtable/_oauth.py

```diff
@@ -159,12 +159,14 @@ async def _refresh_tokens(
             logger.info("Successfully refreshed tokens")
 
             new_credentials = OAuth2Credentials(
+                title="Airtable OAuth2 Credentials",
```

Comment:
`title` is intended for a value we can't generate from the other attributes, which is what we do if `title` is not set